_includes/scenarios/integrate/requests-over-templates.md

```diff
@@ -32,28 +30,33 @@ For simplicity the Request inside the Template only contains an AuthenticationRe
 Even though the Requests are validated during the RelationshipTemplate creation you should not skip this step as it gives you additional information in case of validation errors.
 {: .notice--info}
 
-## Create the Template
+## Create the RelationshipTemplate
 
-Create the Relationship Template on the Templator's Connector. You can do so by calling the `POST /api/v2/RelationshipTemplates/Own` route. Use the following JSON in the Request body:
+Next, [create a RelationshipTemplate]({% link _docs_use-cases/use-case-transport-create-own-relationshiptemplate.md %}) on the Sender Connector, using a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
```

Comment:
Sometimes we write "Sender", sometimes "Sender Connector". Perhaps it would be nicer if we could restructure the sentences so that we could always write "Sender", if possible.